polkadot/node/overseer/src/lib.rs

```diff
@@ -874,11 +870,7 @@ where
 
 	/// Handles a header activation. If the header's state doesn't support the parachains API,
 	/// this returns `None`.
-	async fn on_head_activated(
-		&mut self,
-		hash: &Hash,
-		parent_hash: Option<Hash>,
-	) -> Option<Arc<jaeger::Span>> {
+	async fn on_head_activated(&mut self, hash: &Hash, _parent_hash: Option<Hash>) -> Option<()> {
```

Comment:
I didn't find any reason it is not ok, do you see a problem with it ?

Could probably refactored to be dropped, but I tried to avoid that because it would make it harder to conclude removing jaeger is safe and makes the PR harder to review.